notificationworkerlambda/src/test/scala/com/gu/notifications/worker/WorkerRequestHandlerSpec.scala

```diff
@@ -28,29 +28,46 @@ import scala.collection.JavaConverters._
 class WorkerRequestHandlerSpec extends Specification with Matchers {
 
   "the WorkerRequestHandler" should {
-    "Send one notification" in new WRHSScope {
-      workerRequestHandler.handleRequest(sqsEventShardNotification, null)
+    "Send one content notification" in new WRHSScope {
+      workerRequestHandler.handleRequest(sqsEventShardNotification(breakingNewsNotification), null)
```

Comment:
is a breaking news notification a content notification?

Reply:
🤦‍♂️ good spot